### ISSUE TITLE
ComputeCost could not scale well as more losses are added

### ISSUE THREAD

**Issue by popcornell** [speechbrain/speechbrain#126]
Maybe I am missing something, but right now it seems to me that we have to manually add every new loss function to ComputeCost init with an if statement. 
In this way it could grow excessively large and will be cumbersome to maintain. 

Maybe it could be turned into a wrapper. For most loss function basically we will need only the zero padding masking functionality only.  Otherwise one will have to add BCE, BCEwithlogits, dice_loss etc etc.
